test/integration/trec_fair_2021.py

```diff
@@ -0,0 +1,23 @@
+import re
+import unittest
+from ir_datasets.formats import TrecQrel
+from ir_datasets.datasets.trec_fair_2021 import FairTrecDoc, FairTrecQuery
+from base import DatasetIntegrationTest
+
+
+class TestFairTrec(DatasetIntegrationTest):
+    def test_docs(self):
+        self._build_test_docs("trec_fair_2021",include_count=False)
```

Comment:
Can you copy the output of these _build_text_X runs into here? This way, we will know if the behaviour changes later.